### ISSUE TITLE
Lack of aria-labels on search bar input and clear search button

### ISSUE THREAD

**Issue by petegarvin1** [mbrn/material-table#1884]
When using material table for table filtering, the search bar at the top used to specify the filter has no label. Nor does the button used to clear it. 

This is an accessibility issue and fails when running something like lighthouse over your project. 

Currently I'm having to append these attributes on render which is hardly ideal. 

Can you please add them?

**Comment by oze4** [comment]
@petegarvin1 thank you! We will add it to the list. 

Currently accepting PRs for this!

**Comment by bigianb** [comment]
It looks like the search box has an aria label now but the clear button does not.
